src/Domain/Common/BaseEntity.cs

```diff
@@ -0,0 +1,28 @@
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Domain.Common;
+
+public class BaseEntity
+{
+    public Guid Id { get; set; }
+
+    private readonly List<BaseEvent> _domainEvents = new();
```

Comment:
What is a `BaseEvent`

Reply:
can you look for the BaseEvent file down below this change?